utils/cliutils/commandsflags.go

```diff
@@ -804,6 +808,14 @@ var flagsMap = map[string]cli.Flag{
 		Name:  archive,
 		Usage: "[Optional] Set to \"zip\" to deploy the files to Artifactory in a ZIP archive.` `",
 	},
+	uploadMinSplit: cli.StringFlag{
+		Name:  MinSplit,
+		Usage: "[Default: " + strconv.Itoa(UploadMinSplitMb) + "] Minimum file size in MB to split into ranges when uploading. Require Artifactory with S3 storage.` `",
```

Comment:
...Require Artifactory with S3 storage. --> This option, as well as the functionality of multi-part upload, requires Artifactory with S3 storage.